src/msal-client-credentials-flow/README.md

```diff
@@ -0,0 +1,163 @@
+   "displayName": "active-directory-dotnet-minimal-api-aspnetcore-client-credentail-flow",
+   "description": null,
+   "groupMembershipClaims": null,
+   "identifierUris": [],
+   "isDeviceOnlyAuthSupported": null,
+   "isFallbackPublicClient": null,
+   "notes": null,
+   "publisherDomain": "contoso.onmicrosoft.com",
+   "serviceManagementReference": null,
+   "signInAudience": "AzureADMyOrg",
+   "tags": [],
+   "tokenEncryptionKeyId": null,
+   "defaultRedirectUri": null,
+   "certification": null,
+   "optionalClaims": null,
+   ...
+}
+```
+## Prerequisites
+
+- An Azure Active Directory (Azure AD) tenant. You can [open an Azure account for free](https://azure.microsoft.com/free) to get an Azure AD instance.
+- [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Quickstart: Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the web API.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting    | Value for this sample app                                                    | Notes                                                                                              |
+|---------------------------------:|:------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                          | `active-directory-dotnet-minimal-api-aspnetcore-client-credentail-flow`      | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**       | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**                 | _None_                                                                       | No redirect URI required; don't select a platform.                                                                    |
+| **Client secret**                 | _**Value** of the client secret (not its ID)_                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Configure the web API
+
+1. Open the _~/msal-client-credentials-flow/appsettings.json_ file in your code editor and modify the following values values with those from your [app's registration in the Azure portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-expose-web-apis):
```

Comment:
Is this URL correct? Shouldn't this point to the Azure portal, not the "Expose an API" quickstart? We didn't expose the API, did we?

Related question: No App ID URI needed since we're not exposing and perms via scopes?

Reply:
I think the following resource will do a bit better:

```suggestion
1. Open the _~/msal-client-credentials-flow/appsettings.json_ file in your code editor and modify the following values values with those from your [app's registration in the Azure portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app#register-an-application):
```

Reply:
Yup, that works - that's the one they will have seen if they followed the README steps.